pkg/controllers/provisioning/suite_test.go

```diff
@@ -109,6 +109,17 @@ var _ = Describe("Provisioning", func() {
 		Expect(len(nodes.Items)).To(Equal(1))
 		ExpectScheduled(ctx, env.Client, pod)
 	})
+	It("should continue with provisioning when at least one node pool has instance types", func() {
```

Comment:
```suggestion
	It("should continue with provisioning when one of the provisioners doesn't have resolved instance types", func() {
```

optional: IMO, this makes this more clear to me